tests/cypress/e2e/for-teacher_page/customize_class_page/drag_and_drop.cy.js

```diff
@@ -0,0 +1,77 @@
+import {loginForTeacher} from '../../tools/login/login.js'
+import { createClass } from "../../tools/classes/class";
+
+describe('Back to class button', () => {
+  it('passes', () => {
+    loginForTeacher();
+    cy.wait(500);
+    createClass();
+
+    cy.wait(500);
+
+    cy.get(".view_class").first().click(); // Press on view class button
+    cy.get('#customize-class-button').click(); // Press customize class button
+
+    cy.get('#sortadventures')
+      .children()
+      .should('not.be.visible');
+
+
+    // Click on level 1
+    cy.get("#adventures")
+      .select('1')
+      .should('have.value', '1');
+
+    // Now it should be visible
+    cy.get('#level-1').should('be.visible');
+
+    cy.wait(500)
+
+    // Get the first and second adventure
+    cy.get('#level-1')
+      .children()
+      .eq(0)
+      .invoke('attr', 'adventure')
```

Comment:
What is `invoke` doing here? Is this reading an attribute value using jQuery?

Reply:
Yes, exactly. The invoke function calls the jQuery function to get the attribute. I got that info from here: https://stackoverflow.com/questions/57585785/cypress-get-attribute-value-and-store-in-variable